src/Directory.Build.props

```diff
@@ -11,4 +11,8 @@
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
   </PropertyGroup>
+  <PropertyGroup>
```

Comment:
Or another idea - we can just can move this file to the root and that's it.

Reply:
Yeah, I think that's good. Take a look when you get a chance. And is 6.1.0.0 good for the version number?

Reply:
Yes, I think we need to bump at least the major version for the main package to 6.1.0 and probably keep the other two 6.0.0. And bump the build version for 2 extensions packages.

Reply:
The other two are currently version 6.0.0-alpha1. Just leave that? And I don't know what you mean by the build version.

Reply:
Hmm please bump those two to `6.0.0-alpha2`